### DISCUSSION TITLE
gitee 建个镜像，方便下载

### DISCUSSION THREAD

**Discussion by liuyunli** [NotepadFree/NotepadFree#3] [Ideas]
gitee 能否建个镜像，方便下载

**Comment by SongZihuan** [comment]
是个好主意

**Comment by SongZihuan** [comment]
镜像位于此处：[gitee](https://gitee.com/SuperHuan0630/notepad-free)

**Reply by liuyunli** [reply]
访问受限
另外，不只是代码，release文件也需要

**Reply by SongZihuan** [reply]
刚刚忘记开源了，现在开了。
最新的release已经上传了。
